### ISSUE TITLE
Disable parts of the WP entrypoint functionality

### ISSUE THREAD

**Issue by kfh** [hashicorp/waypoint#940]
Now it's an _all or nothing_ approach where you have to use all the functionality or nothing of it. We have a case where we only would like to use the _url service_  and disable everything else. Would that be possible or are all the parts connected in a way that would make it non trivial?

**Comment by mitchellh** [comment]
Agreed. Thanks for all the awesome enhancement suggestions by the way!

I'm tagging this and we want to be able to flag certain features on/off via some mechanism (env vars perhaps that can be pre-set during the build step?)

**Comment by krantzinator** [comment]
Can you talk a bit about which features you want disabled and the reasoning for doing so? This will help us plan out the documentation around a potential feature here.

**Comment by kfh** [comment]
> Can you talk a bit about which features you want disabled and the reasoning for doing so? This will help us plan out the documentation around a potential feature here.

Yeah, i'll try to remember 😅 As of now we have disabled the entrypoint for all our apps since we aren't really relying on any of the features. 

I think we just missed the opportunity to _pick and choose_ exactly what we needed. In contrast to my original post now the only feature we miss is the possibility to _exec_ directly into a shell on the running container since we are running on _AWS Fargate_ and this isn't supported out of the box. Disabling the url service and keeping the other utility functions i think would be useful in many cases though we found the logs feature underwhelming and just stopped using it and fell back to _AWS Cloudwatch_(which logs are forwarded to anyway out of the box). I've talked more about this [here](https://github.com/hashicorp/waypoint/issues/930).
